flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/table/CalcITCase.scala

```diff
@@ -529,6 +529,27 @@ class CalcITCase(mode: StateBackendMode) extends StreamingWithStateTestBase(mode
     val expected = List("0,0,0", "1,1,1", "2,2,2")
     assertEquals(expected.sorted, sink.getAppendResults.sorted)
   }
+
+  /**
+   * This is an edition of [[CalcITCase.testMap()]] that uses * as one argument of the UDFs.
+   */
+  @Test
+  def testUserDefinedFunctionWithStarParameter(): Unit = {
+    val ds = env.fromCollection(smallTupleData3).toTable(tEnv, 'a, 'b, 'c)
+      .map(Func23('*)).as("a", "b", "c", "d")
+      .map(Func24('*)).as("a", "b", "c", "d")
+      .map(Func1('b))
```

Comment:
Could you also test `where`, `select` too? I remember you tested it in the original test.